docs/userguide/configuring.rst

```diff
@@ -338,6 +343,23 @@ configuration follows.
    `here <https://github.com/Parsl/parsl/issues/941>`_.
 
 
+WorkQueue (CCL ND)
+------------------
+
+.. image:: http://ccl.cse.nd.edu/software/workqueue/WorkQueueLogoSmall.png
+
+The following snippet shows an example configuration for using the Work Queue distributed framework to run applications on remote machines at large. This examples uses the `WorkQueueExecutor` to schedule tasks locally, and assumes that Work Queue workers have been externally connected to the master using the `work_queue_worker` or `condor_submit_workers` command line utilities from CCTools. For more information the process of submitting tasks and workers to Work Queue, please refer to the `CCTools Work Queue documentation <https://cctools.readthedocs.io/en/latest/work_queue/>`.
+
+.. literalinclude::  ../../parsl/configs/wqex_local.py
+
+To utilize Work Queue with Parsl, please install the full CCTools software package within an appropriate Anaconda or Miniconda environment (instructions for installing Miniconda can be found `here <https://docs.conda.io/projects/conda/en/latest/user-guide/install/>`):
+
+.. codeblock:: bash
+    $ conda create -y --name <environment> python=<version>
```

Comment:
does this environment need to be the same environment that parsl needs to be installed in?

Reply:
If Parsl is installed using Pip outside of an environment, then it shouldn't be a problem. If Parsl is installed within a conda environment, then yes, the user would need to have the same environment for both Parsl and Work Queue. I'll add that to the docs